From f4668353266a33e71ab7cdbb99595edbaff03ace Mon Sep 17 00:00:00 2001
From: Matt Turner <mattst88@gmail.com>
Date: Wed, 6 Nov 2019 11:13:56 -0800
Subject: [PATCH 4/4] Provide an empty GLES3/gl3ext.h header

The one in Mesa is empty but appears to come from some Khronos
repository, but gl3ext.h has never existed in the OpenGL-Registry
repository on github. Provide it for backwards compatibility.

Closes: https://gitlab.freedesktop.org/glvnd/libglvnd/issues/195
---
 include/GLES3/gl3ext.h | 9 +++++++++
 include/Makefile.am    | 1 +
 2 files changed, 10 insertions(+)
 create mode 100644 include/GLES3/gl3ext.h

diff --git a/include/GLES3/gl3ext.h b/include/GLES3/gl3ext.h
new file mode 100644
index 0000000..484ea17
--- /dev/null
+++ b/include/GLES3/gl3ext.h
@@ -0,0 +1,9 @@
+#ifndef __gl3ext_h_
+#define __gl3ext_h_
+
+/* 
+ * This file is intentionally empty and is provided for source compatibility
+ * with Mesa.
+ */
+
+#endif /* __gl3ext_h_ */
diff --git a/include/Makefile.am b/include/Makefile.am
index 0c41dc7..c821de1 100644
--- a/include/Makefile.am
+++ b/include/Makefile.am
@@ -31,6 +31,7 @@ GLES2_HEADER_FILES = \
 GLES3_HEADER_FILES = \
 	GLES3/gl31.h \
 	GLES3/gl32.h \
+	GLES3/gl3ext.h \
 	GLES3/gl3.h \
 	GLES3/gl3platform.h
 
-- 
2.24.0

